README.md

```diff
@@ -1,8 +1,50 @@
+
+**Point-of-Sale System**: Implemented a Point-of-Sale system using .NET and MySQL. This system allows users to update, delete, and retrieve data through the .NET WPF frontend. This project highlighted aspects of working with the database and communicating the data retrieved from the constructed queries in a user-friendly format. The app was built on event-driven architecture where CRUD operations would be executed based on the event evoked by the user. It also allows for receipt creation simulation when a new order is created where the virtual receipt displays the current date of the new record, id, list of items along with their assigned prices, and includes the calculated total.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2021-present)**: Developed a variety of different applications using an Agile methodology. Acted as a team lead role to organize and lead daily scrum meetings with the team. Java, Linux, SQL, Docker, and bash scripts were used in developing. I also gained more experience with team collaboration and organization skills related to scrum through the use of Notion for projects to manage my sprint backlog. Leadership was another soft skill gained from the Jarvis opportunity since every new sprint allows a developer to take on the role of team lead and direct the group in scrum meetings. My ability to problem-solve has increased because of the challenges and questions posed by the assignments to ensure that I understand the technical aspects of the projects being developed and can relate the concepts to my prior knowledge.
+
+**Intern, Otekha Health (2021)**: Established new soft skills from working with Otekha Health including team collaboration within a remote agile environment, the ability to learn unfamiliar technologies and understand requirements based on virtual stand-up meeting discussions. Also, the ability to write software documentation such as an agile Software Requirement Specification (SRS) about the system. Refer to Electronic Medical Records System in Highlighted Projects section for system details.
```

Comment:
Developed soft skills from working with Otekha Health including team collaboration within a remote agile environment, the ability to learn new technologies and understand requirements based on virtual stand-up meeting discussions. Also, the ability to write software documentation such as an agile Software Requirement Specification (SRS) about the system. Refer to Electronic Medical Records System in Highlighted Projects section for system details.